publications/abstracts/reservoir-computing-abs/background.tex

```diff
@@ -1,44 +1,49 @@
 \section{Background}
-Variability has been the primary drawback for renewable energy sources like
-wind turbines, solar PV, and solar concentrators, since their inception. This
-flaw has become more pronounced as renewable penetration on the electricity
-grid increased in recent years. Forecasting electricity production from
-renewable sources is therefore important for successful management of power
-systems \cite{kobylinski_high-resolution_2020}. Recent studies have applied
-\glspl{ann}, specifically multi-layer perceptrons, to the task of net load
-forecasting \cite{kobylinski_high-resolution_2020,dutta_load_2017,lee_development_2016}. These studies made short
-term forecasts of 4-6 hours. Nuclear plants
-need accurate forecasts further ahead to facilitate relaxed load following.
-This study will be the first to apply \glspl{ESN} to the task of net load
-prediction.
+
 
 The University of Illinois at Urbana-Champaign is an ideal model system for
-this work because of its diverse energy mix. Previous work has been done to
-characterize this energy grid and optimize the size of a nuclear reactor
-\cite{dotson_optimal_2020}. Due to the degree of wind penetration, the
-University is sometimes
+this work because of its diverse energy mix. Previous work
+characterized this energy grid and optimized the size of a nuclear reactor
+for decarbonization \cite{dotson_optimal_2020}. Due to the degree of wind
+penetration, the University is sometimes
 forced to sell electricity back to the grid operator, MISO, at a loss because
 of overproduction
 from wind energy. Thus, a reliable prediction of electricity production from
 wind and other variable sources will reduce the likelihood of these events.
+Figure \ref{figure:vre} shows the total demand and net demand profiles at the
+\gls{uiuc}.
+Daily variations in electricity demand are reasonably predictable, with a
+usual evening minimum around 40 MW. Weekdays have peaks near 50MW and are
```

Comment:
Do you want to have a space between the number and unit? I think either could work (50MW or 50 MW) as long as you are consistent.